lib/engine/game/g_1894/round/stock.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require_relative '../../../round/stock'
+
+module Engine
+  module Game
+    module G1894
+      module Round
+        class Stock < Engine::Round::Stock
+          protected
+
+          def sold_out_stock_movement(corp)
```

Comment:
are you just copying 1828? or is there a difference

Reply:
Sold out is the same. I changed the sold_out? method though.